src/test/java/com/newrelic/telemetry/opentelemetry/examples/BasicExample.java

```diff
@@ -28,10 +28,10 @@ public static void main(String[] args) throws InterruptedException {
     BatchSpansProcessor spanProcessor = BatchSpansProcessor.newBuilder(exporter).build();
 
     // 3. Add the span processor to the default TracerSdkFactory
-    OpenTelemetrySdk.getTracerFactory().addSpanProcessor(spanProcessor);
+    OpenTelemetrySdk.getTracerProvider().addSpanProcessor(spanProcessor);
```

Comment:
Is a `TracerProvider` the same as a `TracerSdkFactory`? Or should this comment be updated as well?

Reply:
Good call. Will update

Reply:
Updated, can you take another look, @jasonjkeller ?